internal/util/terraform.go

```diff
@@ -174,9 +197,9 @@ func (t *TerraformBinUtil) Init(ctx context.Context, args []string) error {
 }
 
 // Apply will call `terraform apply` with the given vars.
-func (t *TerraformBinUtil) Apply(ctx context.Context, tfVars []string) error {
+func (t *TerraformBinUtil) Apply(ctx context.Context, args []string) error {
 	cfg := ctx.Value(constants.DeployConfig).(*configs.DeployConfig)
-	logFile, err := t.FileSystem.OpenFileWriter(ctx.Value("deployLogFile").(string))
+	logFile, err := t.FileSystem.OpenFileWriter(ctx.Value(constants.DeployLogFile).(string))
```

Comment:
Question -- for `apply`, what happens with your terraform plan? Is there a chance to review it, or is not displayed?

Reply:
If you're an advanced terraform user and know how to review a plan, just use terraform. I've always thought the CLI was for the #restofus. ;)

Reply:
I'd like to get to the point where we can deploy DCE @ optum using the CLI. I can't really think of a good reason why the CLI couldn't handle that, minus a couple minor enhancements like there here and there.
But that can be a conversation for another day.